tests/cli/test_integration_datasets.py

```diff
@@ -24,49 +24,43 @@
 from renku import cli
 
 
+@pytest.mark.parametrize(
+    'doi', [(
+        '10.5281/zenodo.597964', 'y', 'pyndl_naive_discriminat_v064',
```

Comment:
To keep readability up, tuples with so many elements are kinda hard to read. Maybe we could use dict here instead?